capstone-frontend/src/components/BookDescriptionOverlay.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Popover, OverlayTrigger } from 'react-bootstrap';
+
+const popoutDescription = (book) => {
+  let description = book.description || "";
```

Comment:
nit: Use single quotes for string literals (https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes).

Reply:
Fixed!